style.css

```diff
@@ -0,0 +1,181 @@
+body{
+    color: #7069f3;
+}
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+
+.hamburger{
+    position : absolute;
```

Comment:
code formatting is important. There is no space before `:` elsewhere. So it should be same here too. You can use Prettier to auto format too